src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] values = Colors.values();
+        int randomInt = new Random().nextInt(values.length);
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method